analysis_options.yaml

```diff
@@ -3,6 +3,7 @@ analyzer:
   strong-mode:
     implicit-casts: false
   errors:
+    todo: ignore
```

Comment:
does `todo` come up?

Reply:
it hasn't yet in this package but I'm in the habit of adding this line anytime I'm doing a bigger cleanup.